uber/custom_tags.py

```diff
@@ -49,11 +57,15 @@ def event_dates():
 
 @JinjaEnv.jinja_export
 def hour_day_local(dt):
+    if not dt:
+        return ''
     return hour_day_format(dt)
```

Comment:
Say, shouldn't this be `return hour_day_format(dt.astimezone(c.EVENT_TIMEZONE))`? Otherwise it does nothing that hour_day_format doesn't.